### PR TITLE
Fixed a minor problem with redudant calls to `world.getBlockState(pos)`

### PR DESCRIPTION
**Pull request by samtrion**
Fixed a minor problem with redudant calls to `world.getBlockState(pos)`

### CODE REVIEW

**Comment by raoulvdberge** [comment]
Undo the capability cast changes, these are necessary to avoid warnings.

This also doesn't really warrant a changelog entry, because these weren't actual perf issues in the first place ;)

File: src/main/java/com/raoulvdberge/refinedstorage/item/CapabilityProviderEnergy.java [hunk 1]
```diff
@@ -11,12 +11,10 @@
 import javax.annotation.Nullable;
 
 public class CapabilityProviderEnergy implements ICapabilityProvider {
```
Comment (raoulvdberge) [inline src/main/java/com/raoulvdberge/refinedstorage/item/CapabilityProviderEnergy.java]: The changes in this file are dangerous, and involve capability creation semantics. Please undo.

**Comment by raoulvdberge** [comment]
Thank you!

**Comment by Samtrion** [comment]
you are welcome

**Pull request merged**